api_tests/preprint_providers/views/test_preprint_provider_subjects_list.py

```diff
@@ -11,9 +12,9 @@ def create_subject_rules(self):
         '''
         Subject Hierarchy
         +-----------------------------+
-        |                    +-->E    |
-        |      +-------->B+--+        |
```

Comment:
No. That functionality was obviated when we moved to BePress, it just wasn't fixed before.